gpu_id: "1"
debug: true
camera_info: "/azure1/rgb/camera_info"
camera_frame: "azure1_rgb_camera_link"
rgb: "/azure1/rgb/image_raw"
depth: "/azure1/depth_to_rgb/image_raw"
point: "/azure1/points2"
width: 1280
height: 720

is_class_names: ["ikea_stefan_side_left", "ikea_stefan_long", "ikea_stefan_middle", "ikea_stefan_bottom"]
is_config_path: "/home/dyros/Workspace/GIST/assembly_part_segmentation/furniture_part_segmentation/config/rgb.json"
is_weight_path: "/home/dyros/catkin_ws/src/assembly_part_recognition/pretrained_weights/furniture_part/0908_Azure_SIM_v15161718_RGB_only_aug2_epoch_12.tar"
is_thresh: 0.98
roi: [0, 1280, 0, 720]

pe_experiment_name: "exp_group/ikea_stefan"
pe_test_config: "mpaae_maskrcnn_azure.cfg"
pe_class_names: ["ikea_stefan_bottom", "ikea_stefan_long", "ikea_stefan_middle", "ikea_stefan_short", "ikea_stefan_side_left", "ikea_stefan_side_right"]
use_mask_crop: [False, False, False, False] # w.r.t is_class names
use_sameWHcrop: [True, True, True, True] 
model_dir: "/home/dyros/catkin_ws/src/assembly_knowledge/models"
black_borders: True

n_points: 3000
n_iter: 1000
tolerance: 0.005
num_levels: 5



